contentcuration/contentcuration/static/js/edit_channel/import/vuex/importModule.js

```diff
@@ -0,0 +1,25 @@
+var mutations = require('./importMutations');
+var actions = require('./importActions');
+var getters = require('./importGetters');
+
+module.exports = {
+  namespaced: true,
+  state: {
+    // An array of JSON-ified ContentNode Models, which is updated
+    // whenever a user selects a Node to import
+    itemsToImport: [],
+    // start | show_warning | finish | null
+    importStatus: null,
```

Comment:
I think if one wanted to enforce type safety for mutations, they would need to do that at the mutation level by placing a guard within, in this example, `UPDATE_IMPORT_STATUS`, which just necessary boilerplate for (untyped) JS.